src/env.ts

```diff
@@ -0,0 +1,14 @@
+import * as envalid from 'envalid'
+import dotenv from 'dotenv'
+
+dotenv.config({ path: '.env' })
```

Comment:
```suggestion
if (process.env.NODE_ENV === 'test') {
  dotenv.config({ path: 'integration/test.env' })
} else {
  dotenv.config({ path: '.env' })
}
```

Reply:
And obviously create the test.env in the integration folder with the correct defaults for use with the `docker-compose` initiated postgres instance.